gradle/release.gradle

```diff
@@ -11,6 +11,9 @@ releasing {
     git.releasableBranchRegex = "master|release/.+" // matches 'master', 'release/2.x', 'release/3.x', etc.
     git.branch = System.getenv("TRAVIS_BRANCH")
 
+    def buildNo = System.getenv("TRAVIS_BUILD_NUMBER")
```

Comment:
Let's hold off with this just yet... I'm trying to figure out how we can default that in a clean way in the release tools.